src/components/HomePage/Sidebar.js

```diff
@@ -0,0 +1,21 @@
+import './Homepage.css';
+
+export const SideBar = () => {
+    return <aside className='side-wrapper display-flex'>
+        <button className='btn-sideWrapper  btn'>
```

Comment:
for now its ok, but in the future use Link component from the react-router

Reply:
Thank you. I will do the suggested changes.